### PR TITLE
Makes visual adjustments to text

### PR DESCRIPTION
**Pull request by jessicamarcus**
After a conversation with @sebastianebarb :

- Adds a visible underline for inline text links in order to visually differentiate from surrounding body text (WCAG 1.4.1)
- Changes supplementary text labels to a darker gray (WCAG 1.4.3)
- Overrides universal link color in footer (WCAG 1.4.3)

### CODE REVIEW

**Comment by fionawhim** [comment]
Can you walk through the Percy changes with @sebastianebarb? I think there are some that could go either way (bread crumbs, the contact department in section header).

Also, can you look into this one: https://percy.io/CityOfBoston/patterns/builds/1519934/view/101381871/375?mode=diff&browser=firefox&snapshot=101381871

Ideally those links look exactly identical in styling, but the underline is much higher on one of them.

File: stylesheets/components/footer/_footer.styl [hunk 1]
```diff
@@ -47,7 +47,7 @@
       text-transform: uppercase
       text-decoration: none
       font-weight: 500
-      color: $optimistic-blue-light
+      color: $optimistic-blue-light !important
```
Comment (fionawhim) [inline stylesheets/components/footer/_footer.styl]: This is overriding the 311 "Report an issue" yellow link in the footer.

File: stylesheets/base/global/_universal.styl [hunk 2]
```diff
@@ -8,32 +10,14 @@ a
...
-          -0.07em 0 $color-body-background,
-          0 0.05em $color-body-background,
-          0 -0.1em $color-body-background
-;
-
   &:hover
     color: $freedom-red-light
 
+// designed to catch instances where style classes have not been applied to the <a>
+a:not(&[class]):not([title]),
```
Comment (fionawhim) [inline stylesheets/base/global/_universal.styl]: let's talk about this because using "title" feels strange, as does `<li>`

Comment (jessicamarcus) [reply stylesheets/base/global/_universal.styl]: Definitely - I don’t love those selectors at all.

**Comment by jessicamarcus** [comment]
After discussion with Sebastian, he's fine with using the styled underline when it looks best (chrome & safari), and a standard `text-decoration: underline` in other cases. Defaults to standard underline.

File: stylesheets/mixins/styled-underline.styl [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+styled-underline(body-background-color)
+  text-decoration: underline
+
+  &:hover
+    text-decoration: none
+
+  button&
+    background-color: body-background-color
+
+  @supports(-webkit-text-security: circle)
```
Comment (fionawhim) [inline stylesheets/mixins/styled-underline.styl:14]: TIL

**Review by fionawhim — APPROVED** [review]

**Comment by jessicamarcus** [comment]
Abandoning this PR; these changes will occur at a later date.

**Pull request closed without merging**